spec/openstudio/workflow/workflow_analysis_spec.rb

```diff
@@ -597,27 +597,97 @@
     expect(osw_out[:steps].size).to be > 0
     osw_out[:steps].each do |step|
       expect(step[:result]).to_not be_nil
+      # Only the EDA reporting measure step is supposed to have failed
+      if (step[:measure_dir_name] == "Xcel EDA Reporting and QAQC")
+        expect(step[:result][:step_result]).to eq 'Fail'
+      else
+        expect(step[:result][:step_result]).to eq 'Success'
+      end
     end
 
     expected_r = /Peak Demand timeseries \(Electricity:Facility at zone timestep\) could not be found, cannot determine the informati(no|on) needed to calculate savings or incentives./
     expect(osw_out[:steps].last[:result][:step_errors].last).to match expected_r
 
+    # TODO: Temporary comment
+    # Not sure why the in.idf ends up there at the root? Shouldn't it just be
+    # in run/in.idf?
     idf_out_path = osw_path.gsub(File.basename(osw_path), 'in.idf')
     expect(File.exist?(idf_out_path)).to eq true
 
     # even if it fails, make sure that we save off the datapoint.zip
     zip_path = osw_path.gsub(File.basename(osw_path), 'data_point.zip')
+    # TODO: Isn't that the exact opposite of the comment above?
```

Comment:
Please review this!

Reply:
I think we were just making sure that the file did not end up in the root path. I think we want to check this and the case below Line 623.